README.md

```diff
@@ -1,52 +1,130 @@
-    b. click on `Set Up Project` for the repo
-    c. click on `Use Existing Config`
-    d. click on `Start Building`
+[Code](https://github.com/reshufflehq/reshuffle-elasticsearch-connector) |  [npm](https://www.npmjs.com/package/reshuffle-elasticsearch-connector) | [Code sample](https://github.com/reshufflehq/reshuffle/tree/master/examples/elasticsearch)
 
-10. If circle CI checks are all green, you are all set!
+`npm install reshuffle-elasticsearch-connector`
 
-// Keep documentation template below
+This connector uses [Elasticsearch Node JS Client](https://www.npmjs.com/package/@elastic/elasticsearch) package.
 
-## END
 
-# reshuffle-_CONNECTOR_NAME_-connector
+### Reshuffle Elasticsearch Connector
 
-### Reshuffle _CONNECTOR_NAME_ Connector
+This package contains a Reshuffle connector for connecting to [Elasticsearch](https://www.elastic.co/elasticsearch/).
 
-This connector provides <description>.
+The following example add a new entry to an existing index:
+```js
+const { Reshuffle } = require('reshuffle')
+const { ElasticsearchConnector } = require('reshuffle-elasticsearch-connector')
 
-#### Configuration Options:
-```typescript
-interface _CONNECTOR_NAME_ConnectorConfigOptions {
-  foo: string // foo description
-  bar?: number // bar description
+const elasticsearchOptions = {
+  cloud: {
+    id:
+      '<name>:<id>',
+  },
+  auth: {
+    username: '<username>',
+    password: '<password>',
+  },
 }
+
+const app = new Reshuffle()
+const connector = new Elasticsearch(app, elasticsearchOptions)
+
+app.start()
+
+connector.sdk().index({
+  index: 'game-of-thrones',
+  body: {
+    character: 'Daenerys Targaryen',
+    quote: 'I am the blood of the dragon.'
+  }
+})
 ```
 
-#### Connector events
+### Table of Contents
+
+[Setup Cloud elasticsearch](#setup)
+
+[Configuration Options](#configuration)
+
+#### Connector Events
+
+N/A
 
-##### event1 description
-The connector fires this event when ...
+#### Connector Actions
 
-##### event2 description
-The connector fires this event when ...
+[SDK](#sdk) - Retrieve a Node JS Elasticsearch client
+
+[Examples using the SDK](#sdk)
+
+#### <a name="setup"></a>Setup Cloud Elasticsearch
+Setup Elastic cloud:
+
+1. Go to https://cloud.elastic.co/home
+2. Click `Create deployment`
+3. Provide a name and create
+4. Copy the username/password   
+4. Click on your deployment in the deployment list
+5. copy the cloud ID which is <deployment-name>:<key>
+6. See [Configuration Options](#configuration) below on how to connect to this Elasticsearch cloud instance.
+
+#### <a name="configuration"></a>Configuration Options
+
+The Elasticsearch connector takes the same options as the Node JS Elasticsearch client.
+See [Node JS Elasticsearch Client options](https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/client-configuration.html)
+
+As an example, for connecting to elasticsearch cloud, provide options as below:
+
+```typescript
+import ElasticsearchConnector from "./index";
```

Comment:
Do we normally add the import statement in the Readme? I think we shouldn't

Reply:
lol no, good catch, Intellij tried to be smart again